coprocessor/fhevm-engine/host-listener/src/cmd/mod.rs

```diff
@@ -123,6 +128,21 @@ pub struct Args {
         help = "Maximum number of blocks to wait before a block is finalized"
     )]
     pub catchup_finalization_in_blocks: u64,
+
+    #[arg(
+        long,
+        default_value_t = false,
+        requires = "end_at_block",
+        help = "Run only catchup loop without real-time subscription"
+    )]
+    pub only_catchup_loop: bool,
+
+    #[arg(
+        long,
+        default_value_t = 960u64,
+        help = "Sleep duration in seconds between catchup iterations (only with --only-catchup-loop)"
+    )]
+    pub catchup_sleep_secs: u64,
```

Comment:
default could be none, in this case we sleep the time cover by the blocks ? like you go 1000 blocks in the past, you sleep 1000 blocks less a margin ?
ok maybe too complicated.
960 is too big depending on the start option (80 blocks),
E.g. with --start -40, there are holes
Maybe a straight error like, "catchup_sleep_secs" is too big compared to --start -40.
Also ositive index can maybe rejected with only_catchup_loop true.